code_samples/sharding/getting-started/index.js

```diff
@@ -1,4 +1,8 @@
 const { ShardingManager } = require('discord.js');
-const manager = new ShardingManager('./bot.js', { totalShards: 'auto', token: 'your-token-goes-here' });
+const manager = new ShardingManager('./bot.js', {
+	totalShards: 'auto',
```

Comment:
I found out earlier that `totalShards` has a default value of `auto`, so this is rendered as redundant and can be removed, as well as putting the object on a single line again. The guide page codeblock and paragraph(s) should be updated to reflect this as well.